layouts/partials/favicons.html

```diff
@@ -0,0 +1,9 @@
+<link rel="shortcut icon" href="{{ "favicons/favicon.ico" | relURL }}" >
+<link rel="apple-touch-icon" href="{{ "favicons/apple-touch-icon.png" | relURL }}" sizes="180x180">
+<link rel="icon" type="image/png" href="{{ "favicons/favicon-16x16.png" | relURL }}" sizes="16x16">
+<link rel="icon" type="image/png" href="{{ "favicons/favicon-32x32.png" | relURL }}" sizes="32x32">
+<link rel="manifest" href="{{ "favicons/site.webmanifest" | relURL }}">
+<link rel="mask-icon" href="{{ "favicons/safari-pinned-tab.svg" | relURL }}" color="#333333">
```

Comment:
Where does the color #333333 come from?

Reply:
this is the header color
![Screenshot_1](https://user-images.githubusercontent.com/26163841/174974987-213baad9-284f-40c8-9f1b-61a06d513880.png)